### PR TITLE
Add rectruitment messages

### PR DESCRIPTION
**Pull request by sndrs**
adds the message "We are hiring. Ever thought about joining us?
http://developers.theguardian.com/join-the-team.html" to:
- top the html source
- console
- top of the frontend readme

screen shot 2015-05-26 at 16 13 16

screen shot 2015-05-26 at 16 11 26

### CODE REVIEW

**Comment by JustinPinner** [comment]
:cool:

**Comment by piuccio** [comment]
You can get more fancy

http://sandbox.photonstorm.com/console.log/
http://adamschwartz.co/log/

I saw once a console with animation... can't find it again.

File: static/src/javascripts/bootstraps/common.js [hunk 1]
```diff
@@ -548,6 +548,9 @@ define([
             robust('c-run-custom-ab',   modules.runCustomAbTests);
             robust('c-accessibility-prefs',       modules.initAccessibilityPrefs);
             robust('c-international-signposting', modules.internationalSignposting);
+            if (window.console && window.console.log) {
```
Comment (piuccio) [inline static/src/javascripts/bootstraps/common.js]: I'd put this behind a localStorage switch se that we can disable it. ~99% of people looking at the console is already a guardian developer

**Comment by piuccio** [comment]
http://adriancooney.ie/console.image/

**Comment by johnduffell** [comment]
:+1: :+1: :+1: awesome work, ship it!

**Comment by uplne** [comment]
Nice :+1:

**Pull request merged**